app/configuration/Links.scala

```diff
@@ -0,0 +1,10 @@
+package configuration
+
+object Links {
+  val guardianSubscriptionFaqs = LinkConfig(
+    "http://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions",
+    "Frequently Asked Questions"
+  )
+}
+
+case class LinkConfig(href: String, title: String)
```

Comment:
You could expose `guardianSubscriptionFaqs` as a static method on Link (this would allow us to create a single type instead of two).

```scala

case class Link(href: Stirng, title: String)

object Link {
   val guardianSubscriptionFaqs = Link(
      "http://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions",
      "Frequently Asked Questions")
}
```